src/UglyToad.PdfPig/IDrawingProcessor.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using UglyToad.PdfPig.Content;
+using UglyToad.PdfPig.Core;
+using UglyToad.PdfPig.Graphics;
+using UglyToad.PdfPig.Graphics.Colors;
+using UglyToad.PdfPig.PdfFonts;
+
+namespace UglyToad.PdfPig
+{
+    /// <summary>
+    /// IDrawingSystem
+    /// </summary>
+    public interface IDrawingProcessor
+    {
+        /// <summary>
+        /// Init
+        /// </summary>
+        /// <param name="page"></param>
```

Comment:
I'm assuming these will either be filled in or removed in later commits?

Reply:
correct